src/backend/tests/test-data/teams.test-data.ts

```diff
@@ -0,0 +1,26 @@
+import { Team as PrismaTeam } from '@prisma/client';
+import { Team as SharedTeam } from 'shared/src/types/team-types';
+import { flash } from './users.test-data';
+
+const teamId = 'id1';
+const teamName = 'pats';
+const slackId = '69';
+const description = 'best team no cap';
```

Comment:
cap

Reply:
(also i really like this organization to keep the prisma team and the shared team the same)